src/app/zap-templates/zcl/data-model/chip/account-login-cluster.xml

```diff
@@ -27,13 +27,13 @@ limitations under the License.
 
     <command source="client" code="0x00" name="GetSetupPINRequest" response="GetSetupPINResponse" mustUseTimedInvoke="true" optional="false">
       <description>Upon receipt, the Content App checks if the account associated with the client Temp Account Identifier Rotating ID is the same acount that is active on the given Content App. If the accounts are the same, then the Content App includes the Setup PIN in the GetSetupPIN Response.</description>
-      <arg name="tempAccountIdentifier" type="CHAR_STRING"/>
+      <arg name="tempAccountIdentifier" minLength="16" length="100" type="CHAR_STRING"/>
```

Comment:
Fwiw, I am 99% sure minLength has no effect here...  Worth filing a ZAP issue on that, if you think it should work.